website-frontend/src/styles/bootstrap/overrides.scss

```diff
@@ -23,3 +23,14 @@ $position-values: (
   50: 50%,
   100: 100%
 );
+
+$border-radius: 1.5rem;
+
+.card:hover{box-shadow: 3px 3px 10px -4px #00196E;}
+.inner{overflow: hidden;}
+.inner img{transition: all 1.5s ease;}
+.inner:hover img {transform: scale(1.5);}
```

Comment:
Hmm, do you really think this is a good idea? If we do this, for the sake of consistency, all cards on our website should have this style. However, this `transition` makes things look slow...